packages/x-date-pickers/src/internals/hooks/useViews.tsx

```diff
@@ -62,7 +63,7 @@ export interface UseViewsOptions<TValue, TView extends DateOrTimeViewWithMeridie
 }
 
 export interface ExportedUseViewsOptions<TView extends DateOrTimeViewWithMeridiem>
-  extends MakeOptional<Omit<UseViewsOptions<any, TView>, 'onChange'>, 'openTo' | 'views'> {}
+  extends MakeOptional<UseViewsOptions<any, TView>, 'onChange' | 'openTo' | 'views'> {}
```

Comment:
Preferring single source of truth for `onChange` declaration.

Reply:
nice catch!